Exam Labs/2013-12-20 Cloth Cap/lab/as10r1/etc/frr/frr.conf

```diff
@@ -0,0 +1,43 @@
+password zebra
+enable password zebra
+
+log file /var/log/frr/frr.log
+
+debug bgp keepalives
+debug bgp updates in
+debug bgp updates out
+
+router bgp 10
+
+! no bgp ebgp-requires-policy
+no bgp network import-check
+
+neighbor 10.20.0.1 remote-as 20
+neighbor 10.20.0.1 description Router as20r1
+neighbor 10.11.0.11 remote-as 11
+neighbor 10.11.0.11 description Router as11r1
+neighbor 10.14.0.14 remote-as 14
+neighbor 10.14.0.14 description Router as14r1
+
+
+network 10.20.0.0/24
+network 10.11.0.0/24
+network 10.14.0.0/24
+
+neighbor 10.20.0.1 prefix-list mandaTutto out
+neighbor 10.20.0.1 prefix-list rifiutaTutto in
+neighbor 10.11.0.11 prefix-list mandaTutto out
+neighbor 10.11.0.11 prefix-list mandaTutto in
+neighbor 10.14.0.14 prefix-list mandaTutto out
+neighbor 10.14.0.14 prefix-list mandaTutto in
+neighbor 10.14.0.14 route-map localPref in
+
+ip prefix-list rifiutaTutto deny any
+ip prefix-list mandaTutto permit any
+
+route-map localPref permit 10
+match ip address myAccessList
```

Comment:
Please, add indentation to improve readability. (you can find the correct indentation running the command show running-config inside vtysh)